ACN's security and architecture scomposition and harmonization for certification purposes (#1025)
italia/eidas-it-wallet-docs: docs/en/wallet-instance-lifecycle.rst

As part of the activation, the Wallet Provider MUST evaluate the operating system, general technical capabilities of the device, and trustworthiness of the WSCD 
to check compliance with the technical and security requirements, the authenticity and integrity of the installed Wallet Instance, and ensure the keys used for 
key binding resides in a secure WSCD.

> As part of this evaluation, the Wallet Provider MUST classify the WSCD security level of the Wallet Unit as either WL2 or WL3, according to the criteria defined in :ref:`WSCD security levels <wscd-security-levels>`. This classification determines which types of Digital Credentials the Wallet Instance is eligible to receive (PID requires WL3; (Q)EAA requires at least WL2).

Reviewer: This requirement is already given in wallet-attestation-issuance.rst (line 96)
@peppelinux , Does the requirement (PID requires WL3; (Q)EAA requires at least WL2) also apply during the transitional phase before the EUDIW framework is fully operational?

Author: done in 937f1c04759b4e9991d6c0decc2be0ba8223da2e

As part of the activation, the Wallet Provider MUST evaluate the operating system, general technical capabilities of the device, and trustworthiness of the WSCD 
to check compliance with the technical and security requirements, the authenticity and integrity of the installed Wallet Instance, and ensure the keys used for 
key binding resides in a secure WSCD.

.. _wscd-wallet-instance-lifecycle-requirement:

.. note::

>    When the IT-Wallet will be notified EUDIW and the certification scheme will be published and made operative, the Wallet Provider MUST classify the WSCD security level of the Wallet Unit as either WL2 or WL3, according to the criteria defined in :ref:`WSCD security levels <wscd-security-levels>`. This classification determines which types of Digital Credentials the Wallet Instance is eligible to receive (PID requires WL3; (Q)EAA requires at least WL2).

Reviewer 2: ```suggestion
   When the IT-Wallet is notified EUDIW and the certification scheme is published and becomes operative, the Wallet Provider MUST classify the WSCD security level of the Wallet Unit as either WL2 or WL3, according to the criteria defined in :ref:`WSCD security levels <wscd-security-levels>`. This classification determines which types of Digital Credentials the Wallet Instance is eligible to receive (PID requires WL3; (Q)EAA requires at least WL2).
```
